# A version of Blib was provided with the install, and it did not work.
# Removing the -Mblib makes sure that a workin blib version is loaded
# (blib should be installed before installing TermKey)
--- TermReadKey-2.32/Makefile.PL.orig	2014-03-12 19:13:22.000000000 +0100
+++ TermReadKey-2.32/Makefile.PL	2014-07-22 16:55:58.152687354 +0200
@@ -49,10 +49,10 @@
 	$_ .= "
 
 sgtty cchars.h: genchars.pl
-	\$(PERL) -Mblib -I\$(PERL_LIB) genchars.pl
+	\$(PERL) -I\$(PERL_LIB) genchars.pl
 
 distcc: genchars.pl
-	\$(PERL) -Mblib -I\$(PERL_LIB) genchars.pl dist
+	\$(PERL) -I\$(PERL_LIB) genchars.pl dist
 
 ReadKey.c: cchars.h
 
